lib/render/VolumeRenderer.cpp

```diff
@@ -180,6 +180,15 @@ int VolumeRenderer::_paintGL(bool fast)
     return ret;
 }
 
+std::string VolumeRenderer::_getColorbarVariableName() const
+{
+    VolumeParams *vp = (VolumeParams *)GetActiveParams();
+    if (vp->GetValueLong(VolumeParams::UseColormapVariableTag, 0))
+        return vp->GetColorMapVariableName();
+    else
+        return vp->GetVariableName();
+}
+
 void VolumeRenderer::_setShaderUniforms(const ShaderProgram *shader, const bool fast) const
 {
     VolumeParams *vp = (VolumeParams *)GetActiveParams();
```

Comment:
Please use dynamic_cast<> for polymorphic objects.

Reply:
That is not part of this PR but ok.